src/app/components/Upload.svelte

```diff
@@ -14,26 +14,28 @@
   // This problem can also be solved by listening for dragOver
   // However, as of right now, doing so on Chrome when dragging over nested element
   // will cause the drag over effect to be cancelled for a short amount of time, then resume
+  let files: File[] = [];
   let dragEnterCounter = 0;
   let error: Error | null = null;
 
   async function uploadFilesFromDragAndDrop(event: DragEvent) {
     dragEnterCounter = 0;
-    let fileList = filesFromDragEvent(event);
-    await uploadAllFilesOrDisplayError(fileList);
+    const file = filesFromDragEvent(event);
+    await uploadAllFilesOrDisplayError(file);
```

Comment:
Nitpick: it might be more clear to name `file` to `filesFromEvent` or something of the sort, to indicate that the variable is a list of files